src/main/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGenerator.scala

```diff
@@ -19,9 +19,12 @@ object KinesisImportQueryGenerator {
     exaMetadata: ExaMetadata,
     importSpecification: ExaImportSpecification
   ): String = {
-    val kinesisUserProperties = new KinesisUserProperties(
+    var kinesisUserProperties = new KinesisUserProperties(
       importSpecification.getParameters.asScala.toMap
     )
+    if (kinesisUserProperties.hasNamedConnection()) {
+      kinesisUserProperties = kinesisUserProperties.mergeWithConnectionObject(exaMetadata)
```

Comment:
This should not happen here. The values from connection object should be updated just before using (creating Kinesis client) them.

The reason:

Both `IMPORT FROM` script and query generated scripts are visible in the logs. Therefore, remove the key-value string serialized credentials from scripts. And only get them from connection object when needed, just before creating Kinesis client.

Reply:
fixed